tauri-app/src/app/globals.css

```
@@ -471,9 +471,16 @@ body {
 }
 @layer utilities {
   .cm-shortcut-listening {
-    --cm-shortcut-blue: #155dfc;
+    --cm-shortcut-blue: var(--blue600);
     position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)